src/ServiceProvider.php

```diff
@@ -71,6 +71,7 @@ protected function registerCommands(): static
     public function boot()
```

Comment:
Please add this to the register function above so that users do not have to publish the config:

```php
$this->mergeConfigFrom(__DIR__.'/../config/horizon-dashboard.php', 'horizon-dashboard');
```